neural_seq_qa/network.py

```diff
@@ -0,0 +1,331 @@
+import math
+import paddle.v2 as pd
+
+import reader
+
+__all__ = ["training_net", "inference_net", "feeding"]
+
+feeding = {reader.Q_IDS_STR:reader.Q_IDS,
+           reader.E_IDS_STR:reader.E_IDS,
+           reader.QE_COMM_STR:reader.QE_COMM,
+           reader.EE_COMM_STR:reader.EE_COMM,
+           reader.LABELS_STR:reader.LABELS}
+
+def get_embedding(name, input, word_vec_dim, wordvecs):
+    """
+    Defined word embedding
+    
+    :param name: layer name
+    :type name: str
+    :param input: layer input
+    :type input: LayerOutput
+    :param word_vec_dim: dimension of the word embeddings
+    :type word_vec_dim: int
+    :param wordvecs: word embedding matrix
+    :type wordvecs: numpy array
+    :return: embedding
+    :rtype: LayerOutput
+    """
+    return pd.layer.embedding(
+                name=name,
```

Comment:
To follow current PaddlePaddle's style, if the name is not explicitly used, it is set automatically by PaddlePaddle.

Currently, in all the PaddlePaddle's released demos, except the data layer, all the other layers do not need the name.

Reply:
The names of the layers have been removed.